package.json

```
@@ -11,32 +12,84 @@
     "url": "https://github.com/heroku/heroku-connect-plugin/issues"
   },
   "keywords": [
-    "heroku-plugin"
+    "heroku-plugin",
+    "oclif"
   ],
+  "engines": {
+    "node": ">= 20"
+  },
   "license": "Apache-2.0",
   "main": "index.js",
   "scripts": {
-    "test": "CONNECT_ADDON=connectqa mocha && standard",
-    "style-fix": "standard --fix"
+    "build": "rm -rf dist && tsc -b && oclif manifest && oclif readme && mv oclif.manifest.json ./dist/oclif.manifest.json && cp README.md ./dist/README.md",
+    "build:dev": "rm -rf dist && tsc -b --sourcemap && oclif manifest && oclif readme && mv oclif.manifest.json ./dist/oclif.manifest.json && cp README.md ./dist/README.md",
```

```suggestion
    "build:dev": "rm -rf dist && tsc -b --sourcemap && oclif manifest && oclif readme",
```
Same as above, and especially for a dev command.